api/src/workers/base-event-handler.ts

```diff
@@ -55,38 +52,45 @@ abstract class BaseEventHandler {
       // when the gap is too narrow, it returns an error.
       const blockDiff = toBlock - fromBlock;
       if (blockDiff < this.blockThreshold) {
-        console.log("skipping block, blockDiff = ", blockDiff);
+        console.log("Skipping block, blockDiff = ", blockDiff);
         continue;
       }
 
       // do our processing
       let getEventsResult, eventList;
 
       try {
-        // `getEvents` will retrieve all events of the given type within the block height range supplied.
+        // `getEventsResult` will retrieve all events of the given type within the block height range supplied.
         // See https://docs.onflow.org/core-contracts/access-api/#geteventsforheightrange
-        getEventsResult = await send([
-          getEvents(this.eventName, fromBlock, toBlock),
-        ]);
-        eventList = await fcl.decode(getEventsResult);
+        getEventsResult = await Promise.all([
```

Comment:
I'm still confused as to why `Promise.all([...list])` is needed instead of just `Promise.all(list)`